boringtun/src/device/mod.rs

```diff
@@ -637,6 +776,11 @@ impl<T: Tun, S: Sock> Device<T, S> {
                         Some(peer) => peer,
                     };
 
+                    let mut public_key = String::with_capacity(32);
```

Comment:
If hex, then this should be 64, Key is 256bits, one char represents 4 bits, 256 / 4 = 64

Reply:
Done